client/my-sites/theme/main.jsx

```diff
@@ -629,6 +630,27 @@ class ThemeSheet extends Component {
 		);
 	};
 
+	renderReviews = () => {
+		if ( ! config.isEnabled( 'marketplace-add-review' ) ) {
+			return null;
+		}
+		const { name, themeId } = this.props;
+
+		return (
+			<>
+				<AddReviewsModal
```

Comment:
nitpick: use singular

```suggestion
				<AddReviewModal
```

Reply:
Correct 😄 Renaming it

Reply:
@cpapazoglou What do you think about naming it like the file `MarketplaceReviewsModal` ? Or if it is going to live in the `marketplace` folder, we can just name it `ReviewsModal` as [WooCommerce does](https://github.com/Automattic/woocommerce.com/blob/4edc2b7d78e5cff959aa33aa935c61ebed96ff0a/themes/woo/components/reviews/modal/modal.js#L29). Now I'm renaming it I'm thinking that it probably will be used also for edit a review

Reply:
I have renamed it here https://github.com/Automattic/wp-calypso/pull/84480/commits/d169b5b02d50431b10563a9b3678102d5819d661